repositorytools/cli/common.py

```diff
@@ -37,12 +37,15 @@ def _get_parser(self):
 
     def __init__(self):
         self.parser = self._get_parser()
-        self.parser.add_argument("-D", "--debug", action="store_true", dest="debug", default=False,
-                                 help="Print lots of debugging information")
+        self.parser.add_argument("-D", "--debug", action="store_true", help="Print lots of debugging information")
+        self.parser.add_argument("-V", action="store_true", dest="display_version", help="Prints version and exit")
```

Comment:
Not possible sadly, you already used that flag:
https://github.com/stardust85/repositorytools/blob/master/repositorytools/cli/commands/artifact.py#L30